### ISSUE TITLE
[Question]: adding bios to GBC.emu

### ISSUE THREAD

**Issue by Aaron456781** [Rakashazi/emu-ex-plus-alpha#1744]
### Describe your question

Hi, Is there a way to add the bios like there is in the GBA version as I can't see the same option on the gameboy one?

### Add screenshots

Thanks

**Comment by TheBoxGame** [comment]
bios is adding download gba bios and add bios in options

**Comment by Aaron456781** [comment]
Yes I can add the GBA bios in options but I don't get that same option in the GBC emulator and that's the one I'm trying to add

**Comment by TheBoxGame** [comment]
> Tak, mogę dodać BIOS GBA w opcjach, ale nie mam tej samej opcji w emulatorze GBC, a to właśnie ją próbuję dodać

because in gbc there is no option to add bios yet, it is only in gba

**Comment by Aaron456781** [comment]
Ok if that could be added that would be good
